GUI/MainModInfo.cs

```diff
@@ -92,6 +92,7 @@ private TreeNode UpdateModDependencyGraphRecursively(TreeNode parentNode, Module
             }
 
             var node = parentNode == null ? new TreeNode(nodeText) : parentNode.Nodes.Add(nodeText);
+            node.Name = module.name;
```

Comment:
Without the node.Name property I wouldn't know which mod to show on double click. It has nothing to do with refreshing the dependency graph.